backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -157,6 +163,19 @@ class Listing extends BaseEntity {
   })
   applicationPickUpAddressType?: ListingApplicationPickUpAddressType | null
 
+  @Column({ type: "jsonb", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => AddressDto)
+  applicationDropOffAddress: AddressDto | null
```

Comment:
One comment is that at some point I realised that our client generation library will only create proper typing for optional properties marked with `?` sign.

Reply:
But that probably means we should fix it everywhere, not a part of this PR.

Reply:
Added new issue #1444 to address this.